ironic-deployment/ironic/ironic.yaml

```diff
@@ -73,6 +123,24 @@ spec:
           imagePullPolicy: Always
           command:
             - /bin/runironic-conductor
+          readinessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /v1
+              port: 5049
```

Comment:
The ironic-conductor port is 8089: https://docs.openstack.org/ironic/latest/install/install-rdo.html

Reply:
Thanks. replaced.